client/src/components/shared/fallback/GlobalErrorFallback.tsx

```
@@ -0,0 +1,14 @@
+import type { FallbackProps } from "react-error-boundary";
+
+export const GlobalErrorFallback = ({
+  error,
+  resetErrorBoundary,
+}: FallbackProps) => (
+  <div style={{ display: "grid", placeContent: "center", height: "100vh" }}>
+    <div>
+      <h1>Something went wrong</h1>
+      <pre>{error.message}</pre>
+      <button onClick={resetErrorBoundary}>Reload app</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)